### PR TITLE
C4F6 - feature, Provide a list of reported keywords to the admin

### PR DESCRIPTION
**Pull request by omar-mohyeldin**
issue #89 
@Mirna-Yacout @nohahesham @karimngr @AmrARaouf Please review

### CODE REVIEW

File: arability/app/models/keyword.rb [hunk 1]
```diff
@@ -141,7 +142,7 @@ def approve_keyword(keyword_id)
```
Comment (AmrARaouf) [inline arability/app/models/keyword.rb:141]: what's this?

**Comment by omar-mohyeldin** [comment]
This line is auto generated by github to indicat that they have skipped the rest of the model code and jumped to the code I've modified

**Comment by Mirna-Yacout** [comment]
UML reviewed

**Comment by AmrARaouf** [comment]
Doc and testing reviewed

**Comment by nohahesham** [comment]
code reviewed

**Comment by ghost** [comment]
too fast since its just a function xD anyways scenario reviewed

**Pull request merged**